bot/code_review_bot/report/base.py

```diff
@@ -146,6 +146,11 @@ def pluralize(word, nb):
         else:
             comment = ""
 
+        if not all(
+            revision.contains(issue) for issue in issues if issue.is_publishable()
+        ):
+            comment += "(defects might be in the parent stack)\n"
```

Comment:
A new line will occur if I revise it by referring to the review :)
![iss_new_line](https://user-images.githubusercontent.com/29684524/79242242-0666d300-7eaf-11ea-9f7a-e4b964bc4380.png)
If it is okay with you, I think you can use it like this instead of replace.

Reply:
Instead of adding this text, maybe `Code analysis found 2 defects in the diff 42:` could become `Code analysis found 2 defects (some in a parent revision) in the diff 42:`